src/main.ts

```diff
@@ -57,6 +66,26 @@ app.config.globalProperties.$filters = {
       featureValue = featureSplit[2] ? featureSplit[2] : '';
     }
     return featureValue;
+  },
+  getCustomerLoyalty(orderNotes: any, cusotmerLoyaltyOptions: any) {
```

Comment:
If we have not used this utility anywhere, then why we are implementing it in this PR?